src/main/groovy/de/dkfz/roddy/execution/RestExecutionService.groovy

```diff
@@ -39,6 +38,8 @@ import org.apache.http.ssl.SSLContextBuilder
 import org.apache.http.ssl.SSLContexts
 import org.apache.http.ssl.TrustStrategy
 import org.apache.http.util.EntityUtils
+import org.slf4j.Logger
```

Comment:
I am not really sure, why you use a different logger here. Beforehand we used the LoggerWrapper for everything. Do you want to change that?

Reply:
I think for a library the best approach is to use a logging façade, so users of the library are not required to use a specific logging framework.
See also this [answer](https://stackoverflow.com/a/9741118)